org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+            Debug.WriteLine( string.Format( "Comparing {0} Top-Level Domains.", tlds.Length ) );
+
+            // ENews from DataView To List
+            var qry = enewsDV.GetQuery();
+            var eNewsData = qry.Select( item => item ).ToList();
```

Comment:
The `.Select( item => item)` is redundant here. Not sure what this was trying to accomplish. Removing it will yield the same result.